src/suresh/While.java

```diff
@@ -0,0 +1,75 @@
+package suresh;
+
+public class While {
+
+	void Even(int start, int end) {
+		System.out.println( "Even Numbers are:" );
+		int Index =start;
```

Comment:
You can handle this scenario without creating third variable, think on this.
can you use start++ for increment?